web/package.json

```
@@ -10,7 +10,14 @@
         "build": "webpack --config conf/webpack.dev.js",
         "build-prod": "webpack --config conf/webpack.prod.js",
         "test": "jest",
-        "test-coverage": "jest --coverage"
+        "test-coverage": "jest --coverage",
```

Add a space so we know which were added by tailwind.
```suggestion

        "test-coverage": "jest --coverage",
```